.github/workflows/test-and-release.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           # The job status is per matrix item, which is super convenient.
           name: ${{github.job}}-#${{strategy.job-index}}-${{job.status}}-${{join(matrix.*, ',')}}
           path: "**/*.log"
-          retention-days: 7
+          retention-days: 365
```

Comment:
The maximum is 90 days

```suggestion
          retention-days: 90
```

![image](https://github.com/user-attachments/assets/99ebf9e0-6654-4684-b99b-54a679e2ab2d)